bsip-0062.md

```diff
@@ -0,0 +1,44 @@
+ ```
+  BSIP: 0062
+  Title: Close short position 
+  Authors:
+    Stefan Schießl stefan.schiessl@blockchainprojectsbv.com
+    Jerry Liu bitcrab@qq.com
+  Status: Draft
+  Type: Protocol
+  Created: 2019-09-14
+  Worker: TBD
+  ```
+
+# Abstract
+Closing a short position comes with a risk of getting margin called if no spare funds are available to buy the long and reduce the debt to zero. This BSIP introduces a way to put the short position directly on the market.
+
+# Rationale
+Closing a short position with a healthy CR should come with no risk of getting margin called.
```

Comment:
I think that it would be helpful to any reader to expand this rationale to describe the particular process of closing a short position from the [the forum post](https://bitsharestalk.org/index.php?topic=28211.msg328509) this is considered risky.  The general process of closing a short position is not risky in itself.

Reply:
Done